osprey.js

```diff
@@ -30,6 +30,36 @@ exports.addJsonSchema = function (schema, key) {
  * @param  {Object}  opts
  * @return {Promise}
  */
+
+exports.loadFileSync = function (path, opts) {
```

Comment:
This function violates dry principle. It's 90% the same as `loadFile`. It would be better to make a general function say `composeRamlMiddleware(ramlApi)` and use it in both `loadFile` and `loadFileSync`.